### DISCUSSION TITLE
Header images

### DISCUSSION THREAD

**Discussion by Mister-Hope** [Renovamen/blog.zxh.io#34] [Q&A]
I am very intested in how you get every article a header image, including:

- How you get them, are you writing a script to fetch with APIs and download then generate links for them automatically or one by one manually
- Where did you get them, are you using one website or searching among the internet, is the way stable and easy?
- How you ensure image copyright, using images with copyright has law risks, are you using uncopyrighted images or using images which is ownly safe to uncommercial usage. And how can  you ensure that?

I would like to improve banner image support in my blog and my theme, but my original opinion is if I have to search, download and set images manually, it might be a little unacceptable. So I am looking forward to hearing how you did this. Thanks.

**Comment by shoucy** [comment]
我看到作者的博客所使用的头图，有一些是塞尔达的游戏截图，我猜它们可能是玩游戏的副产物。
